central/graphql/resolvers/clusters.go

```diff
@@ -27,63 +27,68 @@ func init() {
 	schema := getBuilder()
 	utils.Must(
 		schema.AddType("PolicyStatus", []string{"status: String!", "failingPolicies: [Policy!]!"}),
+		schema.AddExtraResolvers("Cluster", []string{
+			"alerts(query: String, pagination: Pagination): [Alert!]!",
```

Comment:
Again, not related to your work but I feel we should start ordering these in lexical asc order (with a comment to fellow contributors to maintain it). It would improve the readability immensely.

Reply:
I can address this in follow up PRs